bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/ui/TabDebugger.java

```diff
@@ -324,7 +331,57 @@ private void createGdbServerGroup(Composite parent) {
 			gd.horizontalSpan = ((GridLayout) comp.getLayout()).numColumns - 1;
 			fGdbServerTclPort.setLayoutData(gd);
 		}
+		{
+			Label label = new Label(comp, SWT.NONE);
```

Comment:
We can show this UI only if the esp-config.json file exists, otherwise old functionality should work as-is for backward compatibility.

Reply:
added condition to show UI
https://github.com/espressif/idf-eclipse-plugin/blob/d723544afd5d982c602912e942e0b862c7122f50/bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/ui/TabDebugger.java#L342